imports/ui/Accordian.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/*
+  An accordian for the Statistics component.
+*/
+export default class Accordian extends Component{
+
+  getRandNum(){
+    return Math.floor((Math.random() * 1000000) + 0);
+  }
+    render(){
+      let accId = "accordion"+this.getRandNum();
+      let headingOneId = "headingOne"+this.getRandNum();
+      let collapseOneId = "collapseOne"+this.getRandNum();
+
+      return(
+        <div className="panel-group" id={accId} role="tablist" aria-multiselectable="true">
+          <div className="panel panel-default">
+            <div className="panel-heading" role="tab" id={headingOneId}>
+              <h4 className="panel-title">
+                <a data-toggle="collapse" data-parent={"#"+accId} href={"#"+collapseOneId} aria-expanded="true" aria-controls={collapseOneId}>
+                  {this.props.title}
+                </a>
+              </h4>
+            </div>
+            <div id={collapseOneId} className="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
+              <div className="panel-body">
+              <table className="table table-striped">
+                <thead>
+                  <tr>
+                    <th scope="col">{this.props.col1}</th>
```

Comment:
See the files changed tab for all linter warnings. Let's avoid introducing more linter warnings.

Reply:
Fixed, will make sure Akash (@AkashAryal) knows to check out linter for next time.